proto_proxy.c

```diff
@@ -1041,17 +1099,22 @@ static int proxy_thread_loadconf(LIBEVENT_THREAD *thr) {
         return -1;
     }
 
+    // seed per-thread stats.
+    if (thr->proxy_int_stats == NULL) {
+        thr->proxy_int_stats = calloc(1, sizeof(struct proxy_int_stats));
```

Comment:
Probably very unlikely to happen since this code runs at initialization time, but should we be checking for failed malloc/calloc's here and fail explicitly?

Reply:
Yeah; I can fix that. I'm just lazy since it's nearly impossible for these to fail + at start time.